packages/web/src/components/search/CategorySearch.js

```diff
@@ -777,6 +777,12 @@ class CategorySearch extends Component {
 		);
 	};
 
+	shouldMicRender(showVoiceSearch) {
+		// checks for SSR
+		if (typeof window === 'undefined') return false;
```

Comment:
Quick question @ShahAnuj2610  

Are we not rendering Mic in SSR ?

Reply:
@jyash97 we are. In case of SSR on initial render, the window variable is undefined, so this is just a defensive check for SSR.

Reply:
Cool, yeah just wanted to clarify that we are showing it back or not after didMount.